### PR TITLE
Add search paging

### LINKED ISSUE #1082: Search Results only shows the first 10
**Describe the bug**
When I search for a term that has more than 10 results only the first 10 are displayed. There is no way to load an item that is not in the top 10 results.

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #1082 

Note: Integration testing of search will come in a follow up PR addressing #1089 

Screen Shot 2019-07-31 at 2 13 16 PM

UI Notes:
- Paging includes a first page, previous page, next page, and last page button.
- First, previous, next, and last buttons are disabled as necessary
- Current page is shown as active
- Total result count and current result span are displayed.

### CODE REVIEW

**Comment by aaron-collier** [comment]
A follow up ticket will be written to add "ellipsis" functionality (see: https://react-bootstrap.github.io/components/pagination/#pagination-more) in order to address possible navigation issues with very large result sets, but that can currently be addressed by a higher `searchResultsPerPage` configuration option, so I'm inclined not to include it in this PR.

File: src/components/search/SearchResultsPaging.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    totalHits: json.hits.total,
+    results: json.hits.hits.map(row => ({
+      uri: row._id,
+      title: row._source.title,
+    })),
+  })
+
+
+  const search = (query, queryFrom) => {
+    const uri = `${Config.searchHost}${Config.searchPath}?q=title:${query}%20OR%20subtitle:${query}&from=${queryFrom}&size=${Config.searchResultsPerPage}`
```
Comment (jcoyne) [inline src/components/search/SearchResultsPaging.jsx]: It seems odd that we have this knowledge in here and in `components/search/Search`. Can this functionality be extracted?

File: src/components/search/SearchResultsPaging.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    fetch(uri)
```
Comment (jcoyne) [inline src/components/search/SearchResultsPaging.jsx]: This should be in an action or actionCreator, not a component.

Comment (aaron-collier) [reply src/components/search/SearchResultsPaging.jsx]: Yeah, likely. I was thinking about this as well, but wasn't sure an action/actionCreator was correct. I can extract it pretty easily I think.

**Pull request merged**